src/ClientData/include/ClientData/ModuleData.h

```diff
@@ -78,6 +79,8 @@ class ModuleData final {
   void AddSymbolsInternal(const orbit_grpc_protos::ModuleSymbols& module_symbols,
                           SymbolCompleteness completeness);
 
+  const orbit_client_data::ModuleIdentifier module_identifier_;
```

Comment:
This shouldn't be const. `const` members disable move-constructing.

Reply:
This is gone anyways.